docs/introduction/tasks.md

```diff
@@ -1,148 +1,238 @@
+arguments will produce _N_ tasks, all referring to distinct executions of the
+same simulator to perform the same simulation. In the following snippet, we show
+how to create a task by submitting a SpliSHSPlasH simulation to the API.
 
 ```python
-# Launch a Reef3D simulation as in previous examples.
+# Example of how to run the SplishSplash simulator
 import inductiva
 
+# get some example input files
 input_dir = inductiva.utils.download_from_url(
     "https://storage.googleapis.com/inductiva-api-demo-files/"
-    "reef3d-input-example.zip", unzip=True
-)
-simulator = inductiva.simulators.REEF3D()
-task = simulator.run(input_dir=input_dir)
+    "splishsplash-input-example.zip", unzip=True)
 
-# Get status of the task.
-status = task.get_status()
-print(status) # would be, e.g., "submitted", "started", "success", "failed", "killed"
+# Instantiate the simulator
+splishsplash_simulator = inductiva.simulators.SplishSplash()
 
-# Get the machine type where the task is running.
-machine_type = task.get_machine_type()
-print(machine_type) # e.g., "c2-standard-4", "c2d-standard-16"
+# SPlisHSPlasH requires a .json file as the main config file.
+task = splishsplash_simulator.run(input_dir="splishsplash-input-example",
+                                  sim_config_filename="config.json")
+print(task.id)  # outputs i4ir3kvv62odsfrhko4y8w2an (example id)
+
+# Note that the following call to splishsplash_simulator.run() using the same
+# input arguments would create a new and distinct task
+# task2 = splishsplash_simulator.run(input_dir="splishsplash-example",
+#                                  sim_config_filename="config.json")
+# print (task2.id)  # outputs k9muu1vq1fc6m2oyxm0n3n8y0 (example id)
 
-# Kill a task that hasn't completed yet.
-if status == "submitted" or status == "started":
-    task.kill()
 ```
 
+A task is uniquely identified by an alphanumeric identifier which is unique across
+all tasks submitted to the API by the user. However, a user can create multiple
+`Task` objects that point to the same task by instantiating the `Task` class with
+the same identifier without this resulting in the creation of new tasks on the
+API; they will be different Python objects, but they will refer to the same task
+on the API. This mechanism is useful when the user wants to recreate a `Task`
+object to retrieve information about tasks created in previous sessions.
+
 ```python
+>>> import inductiva
+>>>
+>>> task1 = inductiva.tasks.Task("i4ir3kvv62odsfrhko4y8w2an")
+>>> task2 = inductiva.tasks.Task("i4ir3kvv62odsfrhko4y8w2an")
+>>> print(id(task1))
+4410160112 
+>>> print(id(task2))
+4389863104
+>>> print(task1.id)
+i4ir3kvv62odsfrhko4y8w2an
+>>> print(task2.id)
+i4ir3kvv62odsfrhko4y8w2an  # the same as task1.id
+```
 
-# Block the call until the simulation completes, but
-# without interrupting the remote task if the local process
-# is interrupted. 
-task.wait()
+### Managing a task
```

Comment:
I think some of these entries make more sense in the "How To" section.
Or maybe the title as it is may not be the most appropriate. We are leaving the "Manage X" to the "How To".